Flow.Launcher/SettingPages/Views/SettingsPaneTheme.xaml

```diff
@@ -24,109 +24,351 @@
+                    <Style x:Key="SecondColumnStyle" TargetType="ColumnDefinition">
+                        <Style.Triggers>
+                            <DataTrigger Binding="{Binding ElementName=Editor, Path=IsChecked}" Value="False">
+                                <Setter Property="Width" Value="0" />
+                            </DataTrigger>
+                            <DataTrigger Binding="{Binding ElementName=Editor, Path=IsChecked}" Value="True">
+                                <Setter Property="Width" Value="2*" />
+                                <Setter Property="MinWidth" Value="160" />
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </Grid.Resources>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="8*" />
+                    <ColumnDefinition MaxWidth="250" Style="{StaticResource SecondColumnStyle}" />
+                </Grid.ColumnDefinitions>
+                <!--  Theme Size Editor  -->
+                <Border
+                    Grid.Column="1"
+                    Padding="0 0 0 0"
+                    Background="{DynamicResource Color00B}"
+                    BorderBrush="{DynamicResource Color03B}"
+                    BorderThickness="1"
+                    CornerRadius="0 6 6 0">
+                    <Border.Style>
+                        <Style TargetType="{x:Type Border}">
+                            <Setter Property="Visibility" Value="Visible" />
+                            <Style.Triggers>
+                                <DataTrigger Binding="{Binding ElementName=Editor, Path=IsChecked}" Value="False">
+                                    <Setter Property="Visibility" Value="Collapsed" />
+                                </DataTrigger>
+                            </Style.Triggers>
+                        </Style>
+                    </Border.Style>
+                    <ScrollViewer
+                        ScrollViewer.CanContentScroll="False"
+                        VirtualizingStackPanel.IsVirtualizing="True"
+                        VirtualizingStackPanel.ScrollUnit="Pixel">
+                        <StackPanel>
+                            <Slider
+                                Name="WindowHeightValue"
+                                Margin="12 8 12 8"
+                                VerticalAlignment="Center"
+                                ui:ControlHelper.Header="{DynamicResource SearchBarHeight}"
+                                AutoToolTipPlacement="TopLeft"
+                                IsMoveToPointEnabled="True"
+                                IsSnapToTickEnabled="True"
+                                Maximum="60"
+                                Minimum="20"
+                                TickFrequency="2"
+                                Value="{Binding WindowHeightSize, Mode=TwoWay}" />
```

Comment:
Looks like it's here. but why not just set WindowHeightSize and separate the UI

Reply:
I don't get what are you mean. pardon~?